src/test/suite/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from "path";
+import * as Mocha from "mocha";
+import * as glob from "glob";
+
+export function run(): Promise<void> {
+  // Create the mocha test
+  const mocha = new Mocha({
+    ui: "tdd",
+  });
+  // mocha.useColors(true);
+
+  const testsRoot = path.resolve(__dirname, "..");
+
+  return new Promise((c, e) => {
+    glob("**/**.test.js", { cwd: testsRoot }, (err, files) => {
+      if (err) {
+        return e(err);
+      }
+
+      // Add files to the test suite
+      files.forEach(f => mocha.addFile(path.resolve(testsRoot, f)));
+
+      try {
+        // Run the mocha test
```

Comment:
This is implied by `mocha.run` that you are running mocha

```suggestion
```

Reply:
Yup, these are leftovers from the example, sorry :)